### ISSUE TITLE
TST: test_noncelestial_angular fails with mpldev

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#9817]
In `astropy/visualization/wcsaxes/tests/test_images.py` in `mpldev` job:

```
_____________________ TestBasic.test_noncelestial_angular ______________________
Error: Image files did not match.
  RMS Value: 4.7575043940216615
  Expected:  
    /root/project/results/tmp413camrf/baseline-test_noncelestial_angular.png
  Actual:    
    /root/project/results/tmp413camrf/test_noncelestial_angular.png
  Difference:
    /root/project/results/tmp413camrf/test_noncelestial_angular-failed-diff.png
  Tolerance: 
    0
```

Started failing on Dec 20, 2019.

**Comment by astrofrog** [comment]
This is due to the changes in https://github.com/matplotlib/matplotlib/pull/15970 which seem genuine

**Comment by pllim** [comment]
So, I guess the fix here is to update the reference image?

**Comment by astrofrog** [comment]
Yep! However I'm waiting for Matplotlib to clean up their version number in master (currently it's still 3.1....+...) - we need that to be 3.2....+... so that we can correctly identify when Matplotlib is the latest developer version later than the 3.2 release (which doesn't need this fix). I've been in touch with the MPL developers, so just waiting now.

**Comment by astrofrog** [comment]
GitHub was a little over-ambitious in auto-closing this...

**Comment by astrofrog** [comment]
Once Matplotlib dev's version number is 3.2....+... we can update this line:

https://github.com/astropy/astropy/blob/56997f744987cf1788e9e87a551213f336e232d4/astropy/tests/image_tests.py#L10

as well as the next one.

**Comment by bsipocz** [comment]
> GitHub was a little over-ambitious in auto-closing this...

well, you said the magic spell in that PR :)

**Comment by astrofrog** [comment]
Ah but I didn't say it would fix this, just that it was needed to fix it - in fact I wonder what would have happened if I had said 'does not fix ...' 😆

**Comment by pllim** [comment]
GitHub sees "fix `hashtag_issue_num`" and will close it even if there is a "does not" before it. I had to re-open an issue at least once because of this "feature". 😉

**Comment by pllim** [comment]
Is this still an issue? Can we close?

**Comment by pllim** [comment]
Fixed in #9851
